queue_services/entity-emailer/src/entity_emailer/config.py

```diff
@@ -60,8 +60,6 @@ class _Config():  # pylint: disable=too-few-public-methods
 
     PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 
-    # monitoring
-    SENTRY_DSN = os.getenv('SENTRY_DSN', None)
```

Comment:
Can we remove this key from the environment (once this change is deployed)?

Or perhaps just remove the sentry application from [here](https://github.com/bcgov/lear/blob/f2f7c2caa42973c9949a39d18a6309f24955fbeb/queue_services/entity-emailer/devops/vaults.json#L15)?

Ask Patrick Wei for confirmation that this is the right thing to do.